packages/web-components/fast-foundation/src/menu-item/menu-item.template.ts

```diff
@@ -17,7 +17,49 @@ export const MenuItemTemplate = html<MenuItem>`
         class="${x => (x.disabled ? "disabled" : "")} ${x =>
             x.expanded ? "expanded" : ""}"
     >
-        ${startTemplate}
+        <div part="start-container" class="start-container">
```

Comment:
@khamudom what are the possible implications for this being within the start container here? I'm wondering if this should be separate from start in the template.

Reply:
this is now updated refactored. the inputs and startTemplate are separated and not in one container